client/package.json

```diff
@@ -14,7 +15,7 @@
   "scripts": {
     "start": "react-scripts start",
     "build": "react-scripts build",
-    "test": "react-scripts test",
+    "test": "react-scripts test --transformIgnorePatterns \"node_modules/(?!axios)/\"",
```

Comment:
Is this ignoring test files in node_modules?

Reply:
Here is context on the problem & the fix.
https://github.com/axios/axios/issues/5101